### ISSUE TITLE
How could I run the FCS algorithm result?

### ISSUE THREAD

**Issue by peizhenbai** [kexinhuang12345/MolTrans#11]
Hi @kexinhuang12345 , I have several questions about your FCS algorithm in MolTrans paper:
- What's the dataset that you use to run FCS and get the frequent sub-structure?
- How could I run the FCS algorithm? It seems not be in this repo.
- About the drug SMILE sequence, did you use the canonical SMILE or just the SMILE format provided by the dataset itself?

This is a great job and inspire me a a lot. Many thanks!

**Comment by kexinhuang12345** [comment]
hey, thanks. So FCS is a relatively general algorithm and is discussed in detail in our workshop paper (https://static1.squarespace.com/static/58f7aae1e6f2e1a0f9a56616/t/5e370e2d12092f15876d5753/1580666413389/paper.pdf). It can be applied in any datasets; although we found applying on a larger dataset help identify higher quality substructures. I remembered for MolTrans, we use all drugs in ChEMBL, which can be quickly downloaded here: https://tdcommons.ai/generation_tasks/molgen/#chembl. The FCS algorithm can be found here: https://github.com/kexinhuang12345/ESPF. We use canonical SMILES.
